scout/commands/export/exon.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+import click
+
+from flask.cli import with_appcontext
+
+from scout.commands.utils import builds_option
+from scout.export.exon import export_exons
+from scout.server.extensions import store
+
+LOG = logging.getLogger(__name__)
+
+@click.command('exons', short_help='Export exons')
+@builds_option
+@with_appcontext
+def exons(build):
```

Comment:
I do not think it will be necessary to do the same thing here. This export will only work as input files for chanjo so it is completely up to us what format to use.

Reply:
Ok!